### PR TITLE
feat(Chip)!: add closeable, readonly, and close-icon props

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

### CODE REVIEW

File: src/components/chips/Chip.vue [hunk 1]
```diff
@@ -1,33 +1,72 @@
...
+import { logicNot, logicOr } from '@vueuse/math';
+import { objectOmit } from '@vueuse/shared';
 import { type ContextColorsType } from '@/consts/colors';
 import Icon from '@/components/icons/Icon.vue';
+import { type RuiIcons } from '~/src';
 
 export interface Props {
-  label?: string;
-  dismissible?: boolean;
+  label?: boolean;
```
Comment (kelsos) [inline src/components/chips/Chip.vue:10]: what does the label do now?

Comment (kelsos) [reply src/components/chips/Chip.vue:10]: OK so it makes it not rounded, maybe we can use a different name there to indicated that it makes it non rounded?

Comment (lukicenturi) [reply src/components/chips/Chip.vue:10]: I was following vuetify naming. Okay I think `tile` would be better.

Comment (lukicenturi) [reply src/components/chips/Chip.vue]: updated

**Review by kelsos — APPROVED** [review]
lgtm :) thanks @lukicenturi

File: src/components/chips/Chip.vue (hunk 1 above)
Comment (kelsos) [reply src/components/chips/Chip.vue]: great thank you :) tile sound better yes

**Pull request merged**